posydon/popsyn/io.py

```diff
@@ -146,15 +146,19 @@
     'f_fb': 'float64',
     'spin_orbit_tilt_first_SN': 'float64',
     'spin_orbit_tilt_second_SN': 'float64',
+    'M4': 'float64',
+    'mu4': 'float64',
```

Comment:
I'd suggest to move `M4` and `mu4` to `EXTRA_STAR_COLUMNS_DTYPES`.

Reply:
They are CC parameters, so I put them where I saw the SN_type and other SN parameters too

Reply:
I am not sure I understand the difference between  EXTRA_STAR_COLUMNS_DTYPES and SCALAR_NAMES_DTYPES. Are both at least available for output in a POSYDON run, by adding then in the scalar_names in the .ini file of the run?

Reply:
I'm not sure about all the details of the split-up. I thought to put M4 and mu4 together with the disk masses, because M4 and mu4 only make sense for a single SN prescription. Similarly the disk masses only make sense for BHs.